src/browser/modules/D3Visualization/lib/visualization/renders/init.js

```diff
@@ -134,6 +135,75 @@ const nodeRing = new Renderer({
   onTick: noop
 })
 
+const nodeImage = new Renderer({
+  onGraphChange (selection, viz) {
+    const pattern = selection.selectAll('pattern').data(function (node) {
+      if (node.propertyMap.image_url) {
+        return [node]
+      } else {
+        return []
+      }
+    })
+
+    pattern
+      .enter()
+      .append('pattern')
+      .attr({
+        id (id) {
+          return 'img-fill-' + id.id
+        },
+        patternUnits: 'userSpaceOnUse',
+        x: -39,
+        y: -39,
+        height: 156,
+        width: 156
+      })
+
+      .append('image')
+      .attr('xlink:href', link => link.propertyMap.image_url)
+      .attr({
+        x: 0,
+        y: 0,
+        type: 'image/png',
+        height: 78,
+        width: 78
```

Comment:
same as above, don't hard-code here (see [here](https://github.com/neo4j/neo4j-browser/pull/887/files#diff-da817c60492d350eb701f5f1c3e687c1R154))

Reply:
handy to throw in a (GraSS-controlled) setting for thumbnail opacity here, as in [here](https://github.com/neo4j/neo4j-browser/pull/887/files#diff-da817c60492d350eb701f5f1c3e687c1R155). but more of a feature request than an issue with this code 😛